mypy/test/testcheck.py

```diff
@@ -67,6 +68,11 @@
     'check-newtype.test',
 ]
 
+if hasattr(typed_ast, '__version__'):
+    version_info = tuple(map(int, typed_ast.__version__.split('.')))  # type: ignore
+    if version_info >= (0, 6, 0):
```

Comment:
It'd be better to do a functionality check instead of this version check (especially now that typed_ast no longer exports a version).  You could do so like this:
```py
import typed_ast.ast35
if 'annotation' in typed_ast.ast35.Assign._fields: ...
```